templates/Makefile.erb

```diff
@@ -37,13 +37,13 @@ file-list: file-list-before-build <%= @name %>-project
 
 <%- if @bill_of_materials -%>
 <%= @bill_of_materials %>:
-	mkdir -p '<%= @bill_of_materials.path %>'
-	mv bill-of-materials '<%= @bill_of_materials.path %>'
+	mkdir -p "<%= @bill_of_materials.path %>"
```

Comment:
Should I change it back to single quotes? I vaguely remember something about double quotes being preferable for dealing with spaces... cc/ @MikaelSmith was it you who mentioned that?